### DISCUSSION TITLE
Απόρριψη pull request " Τελική αναφορά/2019169 #1662"

### DISCUSSION THREAD

**Discussion by TaniaStoupi** [courses-ionio/sw#1757] [Q&A] [answered]
Καλησπέρα σας, θα ήθελα να σας ρωτήσω σχετικά με την απόρριψη του αιτήματος ενσωμάτωσης μου. Αναφέρατε πως δεν μπορείτε να το κάνετε δεκτό καθώς δεν βλέπετε κάποια παρουσίαση προόδου και για αυτό τον λόγο δεν μπορείτε να με βαθμολογήσετε. Με βάση την ανακοίνωση σας σχετικά με την παρουσίαση της τελικής προόδου αναφέρετε πως πρέπει να γίνει τελική παρουσίαση προόδου για πλήρη βαθμολόγηση, παρόλα αυτά δεν έχω δει κάπου να αναφέρεται πως στην περίπτωση μη παρουσίασης δεν υπάρχει καθόλου βαθμολόγηση. Επίσης στις οδηγίες σχετικά με το κλείσιμο τον ραντεβού για την αξιολόγηση δεν αναφέρεται κάτι για καθόλου βαθμολόγηση, καθώς ούτε και στις διευκρινίσεις για τα pull request. Θα μπορούσατε να μου εξηγήσετε τον λόγο για τον οποίο δεν έγινε δεκτό το αίτημα μου, ή να μου παραθεσετε το κομμάτι που αναφέρεται στην μη βαθμολόγηση λόγω μη παρουσίασης προόδου; 
Το pull request που αναφέρομαι είναι το εξής: https://github.com/courses-ionio/sw/pull/1662#issue-1698726383
Σας ευχαριστώ πολύ εκ των προτέρων

**Comment by epidrome** [comment, marked as answer]
https://epidrome.github.io/teaching/progress/
